<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-OU0RVOB" failures="1" tests="3" name="Test" time="17.52" errors="0" timestamp="2025-05-05T13:11:46 NZST">
  <testcase classname="testCases.TC001_AccountRegistrationTest" name="verify_account_registration" time="1.703"/>
  <testcase classname="testCases.TC002_LoginTest" name="verify_login" time="11.015">
    <failure type="java.lang.AssertionError" message="Login failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login failed expected [true] but found [false]
at testCases.TC002_LoginTest.verify_login(TC002_LoginTest.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_login -->
  <testcase classname="testCases.TC002_LoginTest" name="skip_test" time="0.0">
    <skipped/>
  </testcase> <!-- skip_test -->
  <testcase classname="testCases.TC003_LoginDataDrivenTest" name="verify_logonDDT" time="0.0">
    <ignored/>
  </testcase> <!-- verify_logonDDT -->
</testsuite> <!-- Test -->
